packages/wrangler/src/publish.ts

```diff
@@ -236,8 +236,12 @@ export default async function publish(props: Props): Promise<void> {
     Or you could pass it in your terminal as \`--compatibility-date ${compatibilityDateStr}\`
 See https://developers.cloudflare.com/workers/platform/compatibility-dates for more information.`);
   }
-
   const triggers = props.triggers || config.triggers?.crons;
+
+  if (config.route === "") {
+    logger.warn("Route/s is being set as an empty string");
```

Comment:
What should the user do if they see this message?

Reply:
That wasn't discussed. I think it should include the `""` will not be published and is being ignored, or something to that effect.